metastats-domain/src/main/groovy/life/qbic/metastats/MetaStatsLogManager.groovy

```diff
@@ -0,0 +1,36 @@
+package life.qbic.metastats
+
+/**
+ * <class short description - One Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
```

Comment:
Insert Longer description
```suggestion
MetaStatsLogManager is intended to correctly handle and log all errors occuring in Metastats and additionally categorizing the errors through a predefined color scheme in the output log depending on the error context
```

Reply:
To keep the formatting (* is missing) I added your suggestions manually: a9fd7aa